microprofile-config/src/test/java/org/jboss/eap/qe/microprofile/config/multiple/deployments/expressions/MultipleWarDeploymentExpressionTestCase.java

```diff
@@ -0,0 +1,199 @@
+                .body(is("8081"));
+    }
+
+    /**
+     * @tpTestDetails Verify that a composed expression is resolved correctly, based on the deployment value, which
+     *                is expanded into the default, that was set via system property.
+     * @tpPassCrit A system property value is overridden by the composed one
+     * @tpSince JBoss EAP XP 6
+     */
+    @Test
+    public void testComposedExpressionIsResolvedProperly() {
+        get(url1.toExternalForm() + "config/" + PROPERTY_NAME_SERVER_ENDPOINT)
+                .then()
+                .statusCode(200)
+                .body(is(PROPERTY_VALUE_SERVER_ENDPOINT_FOO));
+
+        get(url2.toExternalForm() + "config/" + PROPERTY_NAME_SERVER_ENDPOINT)
+                .then()
+                .statusCode(200)
+                .body(is(PROPERTY_VALUE_SERVER_ENDPOINT_BAZ));
+    }
+
+    /**
+     * @tpTestDetails Verify that a multiple expression is resolved correctly, based on the deployment values, which
+     *                are expanded into the default, that was set via system property.
+     * @tpPassCrit A system property value is overridden by the expanded one
+     * @tpSince JBoss EAP XP 6
+     */
+    @Test
+    public void testMultipleExpressionIsResolvedProperly() {
+        get(url1.toExternalForm() + "config/" + PROPERTY_NAME_SERVER_URL)
+                .then()
+                .statusCode(200)
+                .body(is("http://server1:8080/foo"));
+
+        get(url2.toExternalForm() + "config/" + PROPERTY_NAME_SERVER_URL)
+                .then()
+                .statusCode(200)
+                .body(is("http://server2:8081/baz"));
+    }
+
+    public static class GlobalConfigSourceSetupTask implements MicroProfileServerSetupTask {
+
+        @Override
+        public void setup() throws Exception {
+
+            try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+                ModelNode op = new ModelNode();
+                op.get(OP_ADDR).add(SUBSYSTEM, "microprofile-config-smallrye");
+                op.get(OP_ADDR).add("config-source", "test");
+                op.get(OP).set(ADD);
+                op.get("ordinal").set(600);
+                op.get("properties")
+                        .get(PROPERTY_NAME_SERVER_PORT)
+                        .set(PROPERTY_VALUE_SERVER_PORT_DEFAULT);
+                op.get("properties")
+                        .get(PROPERTY_NAME_SERVER_ENDPOINT)
+                        .set(PROPERTY_VALUE_SERVER_ENDPOINT_DEFAULT);
+                op.get("properties")
+                        .get(PROPERTY_NAME_SERVER_URL)
+                        .set(PROPERTY_VALUE_SERVER_URL_DEFAULT);
+                client.execute(op);
+            }
+        }
+
+        @Override
+        public void tearDown() throws Exception {
+
+            try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+                ModelNode op = new ModelNode();
```

Comment:
What about to use more-creaper way? Either by `Operations ops = new Operations(client);` & creaper API, or simply by CLI command like `client.execute("/subsystem=microprofile-config-smallrye/config-source=test:remove");`?